src/Loop/Driver.php

```diff
@@ -134,13 +134,12 @@
     abstract public function enable($watcherId);
 
     /**
-     * Disable a watcher. Disabling a watcher MUST NOT invalidate the watcher.
+     * Disable a watcher. Disabling a watcher MUST NOT invalidate the watcher. Calling this function MUST never fail,
```

Comment:
Should probably fixed there as well. Otherwise you might miss the `never` since only `MUST` is bold.